.github/workflows/tests.yml

```diff
@@ -19,7 +19,7 @@ jobs:
       fail-fast: false
       matrix:
         browser: [chromium, firefox, webkit]
-        os: [windows-latest, ubuntu-latest, macos-latest]
+        os: [windows-latest, ubuntu-latest, macos-12]
```

Comment:
Any reason for this? just curious.

Reply:
The reason was that WebKit should upload large file tests were fixed on macos 12.